src/pkgcheck/checks/codingstyle.py

```diff
@@ -1035,3 +1035,42 @@ class EclassUnquotedVariablesCheck(_UnquotedVariablesCheck):
     def feed(self, eclass):
         for var_name, lines in self._feed(eclass):
             yield EclassUnquotedVariable(var_name, lines=lines, eclass=eclass.name)
+
+
+class ExcessiveLineLength(results.LinesResult, results.Style):
+    """Line is longer than 80 characters."""
+
+    line_length = 120
+    word_length = 110
+
+    @property
+    def desc(self):
+        return f'excessive line length over {self.line_length} {self.lines_str}'
```

Comment:
I wondered about "lines {str.lines_str} exceed {self.line_length}" but I feel like what you currently have is easier to read?

Reply:
I changed it a little. Looks like this now:
```
app-editors/jedit
  ExcessiveLineLength: version 5.4.0-r2: excessive line length (over 120 characters) on line: 29

app-dicts/myspell-de
  ExcessiveLineLength: version 2017.01.12: excessive line length (over 120 characters) on lines: 41, 42
  ExcessiveLineLength: version 2017.01.12-r1: excessive line length (over 120 characters) on lines: 18, 19, 21, 22
```

Reply:
LGTM.